### PR TITLE
create: Avatar component created

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Hey I have created Avatar component

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    height: 100px;
+    width: 100px;
+}
+.avatar-sm{
+    height: 50px;
+    width: 50px;
+}
+.avatar-xs{
+    height: 25px;
+    width: 25px;
```
Comment (ParthParolekar) [inline components/avatar.css]: Use `rem` instead of `px`. It will help you a lot in making your apps responsive.

Comment (arjun-computer-geek) [reply components/avatar.css]: Ok, I will use this. Thanks for reminding me.

**Review by ParthParolekar — APPROVED** [review]

File: components/avatar.css (hunk 1 above)
Comment (arjun-computer-geek) [reply components/avatar.css]: hey, I fixed that can you please approve this?

**Review by Shubhamkr2610 — APPROVED** [review]
Looks good for me. Your code is looking clean .

File: documentation/docs.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+main .component-container .component-preview{
+    border: 2px solid var(--white);
+    margin:20px 0;
+    padding: 20px;
+}
+main .component-container .component-code{
+    width: 100%;
+    height: auto;
+    margin: 30px 0;
```
Comment (partha8) [inline documentation/docs.css]: Use rem instead of px. It will help with responsiveness.

**Review by partha8 — APPROVED** [review]

File: documentation/docs.css (hunk 2 above)
Comment (arjun-computer-geek) [reply documentation/docs.css]: I have changed px to rem. Please approve this.

**Pull request merged**